test/verify/check-networkmanager-basic

```diff
@@ -41,6 +41,17 @@ class TestNetworkingBasic(NetworkCase):
         b.wait_in_text("#networking-graphs", "Transmitting")
         b.wait_in_text("#networking-graphs", "Receiving")
 
+        b.assert_pixels(
+            "#networking", "network-main",
+            ignore=[
+                "#networking-graphs .pf-l-grid",
+                "tr[data-interface=cockpit1] td[data-label=Sending]", "tr[data-interface=cockpit1] td[data-label=Receiving]",
+                "tr[data-interface=eth0] td[data-label=Sending]", "tr[data-interface=eth0] td[data-label=Receiving]",
+                "tr[data-interface=v_cockpit1] td[data-label=Sending]", "tr[data-interface=v_cockpit1] td[data-label=Receiving]",
+                ".cockpit-log-panel"
+            ]
```

Comment:
My first thought was "this is capturing and ignoring too much", but it actually makes sense -- we are testing the page layout (like spacing between cards) as well, which can't hurt. So let's try this, and we can dial it back if it gets in the way somehow. I only hope that we don't get bitten by different scrolling position.

In general, quite a number of these screenshots contain scroll bars -- so whenever firewalld gets a new profile or drops one, or the "Network logs" contain more or fewer messages, the scroll bar slider size will change. It may be better to ignore the scrollbar in the images as well?

Reply:
Indeed, we should be carefull with content that might change with OS updates etc. Which the firewall images indeed are capturing.